bika/lims/browser/resultsimport/autoimportlogs.py

```diff
@@ -0,0 +1,73 @@
+# This file is part of Bika LIMS
+#
+# Copyright 2011-2016 by it's authors.
+# Some rights reserved. See LICENSE.txt, AUTHORS.txt.
+
+from DateTime.DateTime import DateTime
+from bika.lims.browser.bika_listing import BikaListingView
+from bika.lims.utils import tmpID
+from Products.CMFCore.utils import getToolByName
+from bika.lims.exportimport import instruments
+from bika.lims import bikaMessageFactory as _
+from bika.lims.catalog import CATALOG_AUTOIMPORTLOGS_LISTING
+
+import traceback
+from os import listdir
+from os.path import isfile, join
+
+
+class AutoImportLogsView(BikaListingView):
+    """
+    To list all logs...
+    """
+    def __init__(self, context, request):
+        super(AutoImportLogsView, self).__init__(context, request)
+        self.catalog = CATALOG_AUTOIMPORTLOGS_LISTING
+        self.contentFilter = {'portal_type': 'AutoImportLog',
+                              'sort_on': 'Created',
+                              'sort_order': 'reverse'
+                              }
+        self.title = self.context.translate(_("Last Auto-Import Logs"))
+        self.description = ""
+        self.columns = {'ImportTime': {'title': _('Time'),
+                                       'sortable': False},
+                        'Instrument': {'title': _('Instrument'),
+                                       'sortable': False},
```

Comment:
You can use the 'replace_url' and 'attr' attributes for 'Instrument' column. Something like the following:

    'Instrument': {'title': _('Instrument'),
                          'sortable': False,
                          'attr': 'getInstrumentTitle',
                          'replace_url': 'getInstrumentUrl'},